source/dashboards/dashboards.markdown

```diff
@@ -22,7 +22,7 @@ Home Assistant ships with 5 predefined dashboards:
 - Logbook
 - History
 
-Not all of predefined dashboards are listed under {% my lovelace_dashboards title="**Settings** > **Dashboards**" %}. **Map**, **Logbook**, and **History**, are powered by their respective integrations.
+Not all of the predefined dashboards are listed under {% my lovelace_dashboards title="**Settings** > **Dashboards**" %}. **Map**, **Logbook**, and **History**, are powered by their respective integrations.
 
```

Comment:
Map will be a regular dashboard in the next release but we can keep this sentence for now.